src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,76 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private User user;
+    private RegistrationService registration;
+
+    @BeforeEach
+    void setup() {
+        user = new User("gerasimov", "wevrewvre", 18);
+        registration = new RegistrationServiceImpl();
```

Comment:
We dont need to create `RegistrationServiceImpl` before each test. Create it just once